lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -432,4 +432,4 @@
   copy:
     src: magma_resolv.conf
     dest: /etc/systemd/resolved.conf
-  when: preburn and ansible_distribution == "Ubuntu"
+  when: ansible_distribution == "Ubuntu"
```

Comment:
Now we could also remove check for distribution.

Reply:
True but this will be back to preburn soon as I'll build new images with the rnew esolved.conf in it, this is just tmp but yea I'll do some cleanup on this role